p2p/src/test/kotlin/maru/p2p/discovery/MaruDiscoveryServiceTest.kt

```diff
@@ -135,10 +117,10 @@ class MaruDiscoveryServiceTest {
           P2P.Discovery(
             port = 9000u,
             bootnodes = listOf(),
-            refreshInterval = 5.seconds,
+            refreshInterval = 10.seconds,
           ),
       )
-    service = MaruDiscoveryService(keyPair.secretKey().bytesArray(), p2pConfig, forkIdHashProvider, NoOpMetricsSystem())
+    service = MaruDiscoveryService(keyPair.secretKey().bytesArray(), p2pConfig, forkIdHashProvider)
```

Comment:
Minor: Can we please add named args here

Reply:
done